agent/protocol/rocketmq/types.go

```diff
@@ -0,0 +1,27 @@
+package rocketmq
+
+import (
+	"kyanos/agent/protocol"
+)
+
+var _ protocol.ParsedMessage = &RocketMQMessage{}
```

Comment:
Here you are validating the struct respect the interface.

This one and the previous one you do in the other files could be in tests file.

I'm unsure the compiler clean them in the final binary

Reply:
Hey @zxilly

Do you know if we could check that?

Thanks